### DISCUSSION TITLE
Feature Request: Collapse Entry Source by Headings

### DISCUSSION THREAD

**Discussion by andrewfairlie** [craftcms/cms#8470] [Ideas]
On sites with a lot of sections and custom entry sources the left sidebar can sometimes feel a bit overwhelming, particularly on the smaller screens many clients still use.

It would be a great UX addition if the user could collapse or expand the contents of a header on the sidebar to keep their screen focused.

**Comment by kylecotter** [comment]
We'd greatly benefit from this. We're always leveraging the "Heading" and grouping of sections on projects, and we definitely have projects that make the sections list a long scrollable list (especially for admins or ones who can see all sections). Expand/collapse would be nice. Would even throw in the idea of a search box for quick filtering.
